src/mame/drivers/popeye.cpp

```diff
@@ -162,6 +162,7 @@ WRITE8_MEMBER(popeye_state::protection_w)
 static ADDRESS_MAP_START( skyskipr_map, AS_PROGRAM, 8, popeye_state )
 	AM_RANGE(0x0000, 0x7fff) AM_ROM
 	AM_RANGE(0x8000, 0x87ff) AM_RAM
+	//AM_RANGE(0x8800, 0x8c00) AM_RAM // Attempts to initialize this area with 00 on boot
```

Comment:
AM_RANGE is inclusive. Even though it's commented out, did you mean 0x8bff?

Reply:
Ah, yes. Fixed that.